web-apis.md

```diff
@@ -0,0 +1,97 @@
+# Web APIs
+
+## Introduction to Web APIs
+
+Web APIs are interfaces provided by the browser that allow you to interact with the browser and the underlying operating system.
+
+## DOM Manipulation
+
+The Document Object Model (DOM) is a programming interface for web documents. It represents the page so that programs can change the document structure, style, and content.
+
+### Example Code: Changing the Content of an Element
+
+```document.getElementById("demo").innerHTML = "Hello, World!";```
```

Comment:
Would be nice to use the code syntax. An example on how to use it can be found [here](https://stackoverflow.com/questions/33539885/display-highlighted-code-blocks-in-markdown-code-block-in-github-flavored-markdo).
<img width="451" alt="Screenshot 2024-07-02 at 9 20 11 PM" src="https://github.com/sumn2u/learn-javascript/assets/6531541/a82692de-daf0-4b6e-802d-c200c586a947">